src/gui/qgsexpressionbuilderwidget.cpp

```diff
@@ -1296,6 +1310,28 @@ void QgsExpressionBuilderWidget::storeCurrentUserExpression()
   }
 }
 
+void QgsExpressionBuilderWidget::editSelectedUserExpression()
+{
+  // Get the item
+  QModelIndex idx = mProxyModel->mapToSource( expressionTree->currentIndex() );
+  QgsExpressionItem *item = dynamic_cast<QgsExpressionItem *>( mModel->itemFromIndex( idx ) );
```

Comment:
```suggestion
  QgsExpressionItem *item = static_cast<QgsExpressionItem *>( mModel->itemFromIndex( idx ) );
```

Reply:
I directly copied this from other places in the same file, where they used dynamic cast. I will convert all of them

Reply:
Applied on all the files